cypress/integration/publishingFlows/previewArticle.js

```diff
@@ -0,0 +1,73 @@
+describe('Article Editor', () => {
+  describe('v1 Editor', () => {
+    beforeEach(() => {
+      cy.testSetup();
+      cy.fixture('users/articleEditorV1User.json').as('user');
+
+      cy.get('@user').then((user) => {
+        cy.loginUser(user).then(() => {
+          cy.visit('/new');
+        });
+      });
+    });
+
+    it('should preview blank content of an article', () => {
+      cy.findByTestId('article-form').as('articleForm');
+
+      cy.get('@articleForm')
+        .findByText(/^Preview$/i)
+        .click();
+    });
+
+    it(`should show error if the article content can't be previewed`, () => {
+      cy.findByTestId('article-form').as('articleForm');
+
+      // Cause an error by having a non tag liquid tag without a tag name in the article body.
+      cy.get('@articleForm')
+        .findByLabelText('Post Content')
+        .type('{%tag %}', { parseSpecialCharSequences: false });
+
+      cy.get('@articleForm')
+        .findByText(/^Preview$/i)
+        .click();
+
+      cy.findByTestId('error-message').should('be.visible');
+    });
+  });
+
+  describe('v2 Editor', () => {
+    beforeEach(() => {
+      cy.testSetup();
+      cy.fixture('users/articleEditorV2User.json').as('user');
+
+      cy.get('@user').then((user) => {
+        cy.loginUser(user).then(() => {
+          cy.visit('/new');
+        });
+      });
+    });
+
+    it('should preview blank content of an article', () => {
+      cy.findByTestId('article-form').as('articleForm');
+
+      cy.get('@articleForm')
+        .findByText(/^Preview$/i)
+        .click();
```

Comment:
**Question:** Is this test missing a `should` matcher to ensure no errors are visible? Something like the other test below but along the lines of

```js
// Notice how I don't know the exact syntax of the "should_not" :)
cy.findByTestId('error-message').should_not('be.visible');
```

Without knowing much about Cypress it feels like if an error does bubble up again we won't be able to catch it but I may be totally wrong

Reply:
Good catch. I'll add a check to ensure the error is not visible in the successful path.

Reply:
Done.